.stylelintignore

```diff
@@ -0,0 +1 @@
+media/css/careers/*
```

Comment:
Best to add this to [.stylelintrc](https://github.com/mozilla/bedrock/blob/master/.stylelintrc#L39) instead, as it's what the pre-commit hook uses for reference.

Reply:
Thanks